examples/061_TopLevelCantBeImplicitID.scala

```diff
@@ -1,6 +1,8 @@
-// I can't seem to reproduce this one. I see in `Checking.scala` that is should
-// be it if sym.owner.is(Package), but when playing around that doesn't seem to
-// ever be true
-// INCOMPLETE
+// LEGACY In the past top level implicit were not allowed. This is however no longer the case as of https://github.com/lampepfl/dotty/pull/5754
+// START
 @main def TopLevelCantBeImplicitID = ()
 
+import scala.language.implicitConversions
+implicit def intToBoolean(x: Int): Boolean = x != 0
```

Comment:
Sorry if I wasn't clear in my message. I mean everything except the 
```
implicit val foo: String = "foo"
```
can be removed so then this would just be

```scala
@main def TopLevelCantBeImplicitID = ()
implicit val foo: String = "foo"
```
Since that's the most minimal to illustrate the top level implicit right?

Reply:
Ah, right! I'll remove the `implicit def` part 👍

Reply:
Even the import isn't needed though right? Since it's not doing anything?

Reply:
Oh, yeah, it's now not needed!